IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json

```diff
@@ -5788,6 +5788,434 @@
 						}
 					],
 					"description": "This folder contains examples from the [Organization Management (REST API Example)](https://help.acumatica.com/Help?ScreenId=ShowWiki&pageid=afa51ca5-37ae-4d7e-aa36-761779ca9320) chapter of the Integration Development Guide."
+				},
```

Comment:
I do not see any descriptions added, such as "This folder contains examples from ..." and "This example shows ..."